cmd/lncli/commands.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bufio"
 	"bytes"
 	"context"
+	"crypto/rand"
```

Comment:
Commit message doesn't match at all? Super bare in general.

Reply:
Yes, very bare, not good. Added more info.